The idea of this patch is to allow mips, a multiple ABI architecture
to have libc headers installed when the host compiler doesn't recognise
the ABI.

--- glibc-2.3.5/sysdeps/mips/mips32/Makefile.orig	Wed Jun 22 01:22:38 2005
+++ glibc-2.3.5/sysdeps/mips/mips32/Makefile	Wed Jun 22 01:23:59 2005
@@ -1,3 +1,3 @@
-ifeq ($(filter -mabi=32,$(CC)),)
-CC += -mabi=32
-endif
+#ifeq ($(filter -mabi=32,$(CC)),)
+#CC += -mabi=32
+#endif
